libs/datamodel/connectors/datamodel-connector/src/walker_ext_traits.rs

```diff
@@ -47,25 +47,6 @@ impl<'db> DefaultValueExt<'db> for DefaultValueWalker<'_, 'db> {
     }
 }
 
-pub trait RelationFieldWalkerExt<'ast> {
-    fn final_foreign_key_name(self, connector: &dyn Connector) -> Cow<'ast, str>;
-}
-
-impl<'ast> RelationFieldWalkerExt<'ast> for RelationFieldWalker<'ast, '_> {
-    fn final_foreign_key_name(self, connector: &dyn Connector) -> Cow<'ast, str> {
-        self.explicit_mapped_name().map(Cow::Borrowed).unwrap_or_else(|| {
-            let column_names: Vec<&str> = self
-                .fields()
-                .into_iter()
-                .flatten()
-                .map(|f| f.final_database_name())
-                .collect();
-            let table_name = self.model().final_database_name();
-            ConstraintNames::foreign_key_constraint_name(table_name, &column_names, connector).into()
-        })
-    }
-}
```

Comment:
This code is is a worse, duplicate version of what is in `InlineRelationWalker`, so it can be cleaned up away.